e107_plugins/download/download_shortcodes.php

```diff
@@ -1148,7 +1148,7 @@ function sc_download_back_to_list($parm = null)
 		$url = e107::url('download', 'category', $this->var);
 		// e_PLUGIN_ABS."download/download.php?action=list&id=".$this->var['download_category']
 
-		$title = "Back to [x]";
+		$title = LAN_dl_80;
```

Comment:
please use 

    $title = defset('LAN_dl_80', "Back to [x]");  // less fatal errors when the LAN is missing under PHP 8.x

Reply:
Ok, but that will simply "defeat" the hardcoded language in code removal ideia....

It could be better to load the default complete Eng Lan files if the lan is not found...

But ok...